docs/guide/index.asciidoc

```diff
@@ -68,6 +68,20 @@ open source Kibana dashboards that come with APM Server.
 We designed Elastic APM to run on anything from a regular laptop to thousands of machines,
 and it's easy to get started.
 
+[[separate-component]]
+== Why is APM Server a separate component?
+
+The APM Server is kept as a separate component for the following reasons:
+
+* Agents can offload their heavy lifting tasks to the APM Server which can be scaled independently,
+  since it is a separate component thereby reducing the load on your app and by keeping the agents as light as possible.
```

Comment:
```
Agents can offload their heavy lifting tasks to the APM Server thereby reducing the load on your app. It helps keep the agents as light as possible and since the APM Server is a separate component, it can be scaled independently.
```